lessons/lesson4.md

```diff
@@ -0,0 +1,145 @@
+## Lesson 4: Ensure Events are published
+
+There are three ways one can consume event
+* Using Journaling API 
+* Using runtime action 
+* Using webhook URL
+
+
+### Option 1: Using Journaling API to consume events 
+For enterprise developers, Adobe offers journaling to consume events. The Adobe I/O Events Journaling API enables enterprise integrations to consume events according to their own cadence and process them in bulk. Unlike webhooks, no additional registration or other configuration is required; every enterprise integration that is registered for events is automatically enabled for journaling. Journaling data is retained for 7 days 
+
+After you fire event, you should be able to verify your event through journaling `UNIQUE API ENDPOINT` you get from console by follow below instruction
+[Journaling api](https://www.adobe.io/apis/experienceplatform/events/docs.html#!adobedocs/adobeio-events/master/intro/journaling_api.md)
+You could use `Curl` command or `POSTMAN` to call this journaling `UNIQUE API ENDPOINT` to see your fired event.
+Or you can use [Custom event SDK](https://github.com/adobe/aio-lib-events/) to calling Journaling API to retrieve your event
+
+
+### Option 2: Using runtime action
```

Comment:
So this explains a Runtime + Slack webhook integration right ?  I'm not sure if this is necessary in this lab which is focusing on Custom Events.